ui/appui-react/src/appui-react/keyboardKey/KeyboardKey.ts

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+/** Enumeration for Function Keys
+ * @internal
+ */
+export enum FunctionKey {
```

Comment:
Instead of redefining those in each package, use the one in core-react (leave it internal, but do not duplicate the defintion)